Add Odoo app

nardellu opened the pull request:
Here's the first version of ns8-odoo app. (Package version: 16.20240509.0)

compgeniuses commented:
hello @nardellu thank you for this wonderful addition to nethserver 8.

Could you kindly first Make a Community announcement here https://community.nethserver.org/c/app/26 that way, other community members can help test and be aware of your addition to the community before its added into the nethforge repo.

thank you

nardellu commented:
@DavidePrincipi thanks for private clarifications. 

- We removed unnecessary firewall rules.
- We fixed some bugs found during internal tests. 
- Changed to a better semantic version, now: 1.0.0
- About manual, we are working on and tring to explain as much as possibile. We will talk soon!

@compgeniuses 

- We will make the announcement as soon as we have finished the manual. I believe in the short term. Thank you
